Sources/Stevenson/Errors.swift

```diff
@@ -76,12 +76,22 @@ extension GitHubService: FailableService {
 }
 
 extension JiraService: FailableService {
-    struct ServiceError: Swift.Error, Decodable, Debuggable {
-        let message: String
-        let identifier: String = "JiraService"
+    public struct ServiceError: Swift.Error, Decodable, Debuggable {
+        // See https://developer.atlassian.com/cloud/jira/platform/rest/v3/#status-codes for schema
+        public let errorMessages: [String]
+        public let errors: [String: String]
+        public let identifier: String = "JiraService"
 
-        var reason: String {
-            return message
+        public var reason: String {
+            let allErrors = errorMessages + errors.sorted(by: <).map { "\($0): \($1)" }
```

Comment:
Nit: I'm a great fan of early returns, this is very minor but you could do:
```
guard errorMessages.count > 0 || errors.cont > 0 else {
  "Unknown error"
}
```

Reply:
tbh I thought about it but for those cases you're supposed to use `.isEmpty` not `.count` for performance reasons, which means that the expression would be `guard !errorMessages.isEmpty || !errors.isEmpty else ...` which feels kind of hard to parse/read don't you think?

Reply:
Good point, my idea with early return was to save on unnecessary `.map` but indeed using `.count` would not give us any savings.